### DISCUSSION TITLE
3.0版本挂载各种网盘报错

### DISCUSSION THREAD

**Discussion by arbo11** [AlistGo/alist#1646] [Q&A]
在n1盒子上用Docker部署3.0版本，添加各种存储都提示Failed init storage but storage is already created，请教大家是什么原因？
![image](https://user-images.githubusercontent.com/113317056/189554145-211a5e93-5255-41a2-a13b-dbba645a08fb.png)

**Comment by huangteng135** [comment]
我也是一样的，openwrt里面docker安装alist，添加网盘报错。不清楚什么原因。。。。

**Reply by asdljl** [reply]
试下网络host模式,我刚开始n1也是这样，后面试了host模式，就好了

**Comment by stanfu0701** [comment]
同原因 openwrt docker 安装3.0版本 添加不了网盘

**Reply by asdljl** [reply]
试下网络host模式,我刚开始n1也是这样，后面试了host模式，就好了

**Comment by asdljl** [comment]
试下网络host模式,我刚开始n1也是这样，后面试了host模式，就好了
